webservices/common/models/presidential.py

```diff
@@ -66,22 +66,14 @@ class PresidentialBySize(db.Model):
 class PresidentialByState(db.Model):
 
     __table_args__ = {'schema': 'public', 'extend_existing': True}
```

Comment:
@fec-jli I think we can remove the `'extend_existing': True` from the table args - I just added that so it wouldn't yell at me about re-using the RAD Analyst model.